src/theia/extension.theiaFF.ts

```diff
@@ -0,0 +1,181 @@
+import { sleep } from "@zowe/cli";
+/*
+* This program and the accompanying materials are made available under the terms of the *
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at *
+* https://www.eclipse.org/legal/epl-v20.html                                      *
+*                                                                                 *
+* SPDX-License-Identifier: EPL-2.0                                                *
+*                                                                                 *
+* Copyright Contributors to the Zowe Project.                                     *
+*                                                                                 *
+*/
+
+import { Builder, By, Key, until, Button } from "selenium-webdriver";
+// tslint:disable-next-line: no-submodule-imports
+import * as firefox from "selenium-webdriver/firefox";
+import { TheiaLocator, DatasetsLocators, UssLocators, JobsLocators } from "./Locators";
+
+const WAITTIME = 30000;
+const SleepForTwoSec = 2000;
```

Comment:
See other comments

Reply:
Gave same variable name in all the test files